### ISSUE TITLE
'elementor/element/post/document_settings/after_section_start',  Action stopped  working from 2.7

### ISSUE THREAD

**Issue by latheeshvmv** [elementor/elementor#9080]
## Description
add_action('elementor/element/post/document_settings/after_section_start', this was all good till 2.68 . But after this is not working.  

## Steps to reproduce

```
add_action('elementor/element/post/document_settings/after_section_start', 'function_to_test', 20 , 2 );

function_to_test(\Elementor\Core\DocumentTypes\Post $page){
$page->start_controls_section(
			'content_section',
			[
				'label' => __( 'Content', 'plugin-name' ),
				'tab' => \Elementor\Controls_Manager::TAB_CONTENT,
			]
		);

		$page->add_control(
			'widget_title',
			[
				'label' => __( 'Title', 'plugin-domain' ),
				'type' => \Elementor\Controls_Manager::TEXT,
				'default' => __( 'Default title', 'plugin-domain' ),
				'placeholder' => __( 'Type your title here', 'plugin-domain' ),
			]
		);

		$page->end_controls_section();
}
```

## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x ] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

**Comment by latheeshvmv** [comment]
i noticed that do_action( 'elementor/documents/register_controls', $this );  works on the 2.7 but not on 2.68.  No fallbacks?  Please keep the filters and actions intact

**Comment by latheeshvmv** [comment]
Something is really wrong `  add_action('elementor/element/wp-post/document_settings/after_section_start'` seems to be the new one now ..is wp the prefix? 

![image](https://user-images.githubusercontent.com/44595562/64923647-630aee00-d7f9-11e9-8784-7e1839642c29.png)

after changing the settings-> page refreshes and then setting become empty and I have to click the setting again  all happening on 2.7 update

**Comment by latheeshvmv** [comment]
Any update on this bug? The action issue seems to be got fixed by latest version but the empty settings problem sill there. As the screen refreshes the settings become empty

**Comment by latheeshvmv** [comment]
is there any changes to 
```
  elementor.settings.page.addChangeCallback( 'ggowl_d_custom_post_selector', function( newValue ) {
            elementor.saver.update( {
                onSuccess: function() {
                    elementor.reloadPreview();
                    elementor.once( 'preview:loaded', function() {
                        elementor.getPanelView().setPage( 'page_settings' );
                    } );
                }
            } );
        } );

```

**Comment by latheeshvmv** [comment]
elementor.getPanelView().setPage( 'page_settings' ); this results in the problem i think

**Comment by VadymPogorelov** [comment]
This is because the developers have once again changed the name of the element (panel) with Page Settings: first it was called 'page-settings', then 'post', now 'wp-post'. 
I have been developing themes for WordPress for many years and worked with different Page Builders (such as Visual Composer, Site Origin Panels, King Composer) and none of them allowed themselves to change the rules of the game so often! I wish the developers of "Elementor" took into account that they do not write a usual plugin. They write a Page Builder, for which other developers will write addons. And with each such change of names that are used in actions and filters or names and class structures, they doom many other developers to problems with clients who suddenly have something stopped working and urgent release of updates to their themes and plugins due to lost compatibility! Over several years of use WPBakery PageBuilder (aka Visual Composer) we haven't been as many forced updates, how much we had to do over the past year, the use of Elementor!
We do not in any way regret the transition - Elementor is really the best PageBuilder that is on the market today, but we would like its creators to pay more attention to backward compatibility when releasing new updates.
I apologize for my English, but I hope I will understand and hear the developers of "Elementor".

**Comment by VadymPogorelov** [comment]
And until for compatibility with different versions was in Elementor I forced to do so:

```
// Add tab with theme-specific Page Options to the Page Settings
//---------------------------------------------------------------
if ( ! function_exists( 'basekit_elm_add_page_options' ) ) {
	add_action( 'elementor/element/after_section_end', 'basekit_elm_add_page_options', 10, 3 );
	function basekit_elm_add_page_options( $element, $section_id, $args ) {
		if ( is_object( $element ) ) {
			$el_name = $element->get_name();
			if ( in_array( $el_name, array( 'page-settings', 'post', 'wp-post' ) ) && 'section_page_style' == $section_id ) {
				...
```

**Comment by latheeshvmv** [comment]
@VadymPogorelov  Yeah I agree!. They have marked it as a bug but still no update, It's been 10 days. No wonder why people are moving to Divi. Very good support and coding standards, documentation. Elementor Developers has follow some standards if they want to grow. 

They have another action for that but there is some bug with  `elementor.getPanelView().setPage( 'page_settings' );`

**Comment by meceware** [comment]
Hey,

I solved this problem via calling:

`$e.route('panel/page-settings/fp-settings' ) );`

Somehow this works. 

Cheers,
